lib/api/v1/routers/router.go

```diff
@@ -21,4 +21,5 @@ func ComebineRoutes(engine *gin.Engine) {
 	authenticatedV1.Use(middlewares.HandleAuthenticatedRequest())
 
 	authenticatedV1.POST("/keywords/upload", controllers.KeywordsController{}.Upload)
+	authenticatedV1.GET("/keywords", controllers.KeywordsController{}.List)
```

Comment:
```suggestion
	authenticatedV1.GET("/keywords", controllers.KeywordsController{}.List)
	authenticatedV1.POST("/keywords/upload", controllers.KeywordsController{}.Upload)
```

We can order it by route name (the root `keywords` before any sub-route). Or by CRUD conventions wich order follows the table:

![image](https://github.com/markgravity/golang-ic/assets/77609814/64cfbeb9-2fcf-415b-8840-f72b347ba16a)